specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-07-01-preview/registries.json

```diff
@@ -637,6 +637,15 @@
           "$ref": "#/definitions/ManagedResourceGroupSettings",
           "x-nullable": true
         },
+        "syndicatedRegistries": {
+          "description": "Dictionary of syndicated registries. Key is Guid and value is a SyndicatedRegistry",
+          "type": "object",
+          "additionalProperties": {
```

Comment:
This is the cause of LintDiff error `AvoidAdditionalProperties`.  It wasn't from a common type.  Are you sure you still want to ignore it?

Reply:
Could you please help show us how to resolve this error? Thank you!

Reply:
We are seeing other similar implementations in existing code. We tried some effort to get it passed, but no luck.

Reply:
I added the suppressing into readme file in my PR: https://github.com/Azure/azure-rest-api-specs/pull/31634/files

Reply:
You should contact the ARM team regarding this.  I would have expected them to inspect this during the ARM review.

https://aka.ms/armofficehoursinfo